test/views/section2/AuthorisationProcedureCodeChoiceViewSpec.scala

```diff
@@ -43,25 +43,25 @@ class AuthorisationProcedureCodeChoiceViewSpec extends PageWithButtonsSpec with
       messages must haveTranslationFor("declaration.authorisations.procedureCodeChoice.radio.1040")
```

Comment:
I noticed that the expander's content is not tested at all. Considering that contents are not tested by the acceptance tests
we should test in our unit tests that the page is displayed according to the requirements.

Just testing that the `messages` file contains the expected messages is not enough (and not useful if you ask me if we verify the presence of all headings and paragraphs).